rllib/agents/ddpg/ddpg_torch_policy.py

```diff
@@ -274,7 +274,7 @@ def setup_late_mixins(policy, obs_space, action_space, config):
     optimizer_fn=make_ddpg_optimizers,
     validate_spaces=validate_spaces,
     before_init=before_init_fn,
-    after_init=setup_late_mixins,
+    before_loss_init=setup_late_mixins,
```

Comment:
This is the more accurate kwarg to use (torch did not have a loss init step before, so this is new). The old after_init still works the exact same and thus this does not cause an API-break.